docs/wiki/[Examples]-Deploy-Custom-Management-Landing-Zone.md

```diff
@@ -0,0 +1,131 @@
+- Add parameters to set location and resource tags
+- Create custom configurations for Log Analytics and ASC
+
+
+The module allows for further configuration of log analytics and azure security center by setting the `configure_management_resources` variable.
+This configuration requires three mandatory variables:
+     location: where the resource group will be deployed
+     advanced:  [tbd]
+     tags: add any specific tag that the resources permit
+
+For this configuration you must pass in both log analytics and security center. If you want to disable
+either or both of these resources, set the enabled flag to *false*.
+
+```hcl
+    settings = {
+      log_analytics = {
+        enabled = false
+     }
+```
+If you've already deployed management resources, this will allow you to enable or disable specific parameters.
+Every parameter set to true will create new principal id's that will force replacement in place of the existing
+roles and policies to reassign them to the log analytics and sentinel resources specified. Likewise, if set to false, existing configurations will be removed.  
+
+If location is not specified, the resources will default to *eastus*
+```hcl
+    terraform {
+      required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = ">=2.46.1"
+        }
+   
+     }
+    }
+
+    provider "azurerm"{
+    features {}
+    }
+
+    #Pull current Tenant ID from connection settings and store to data source
+    
+    data "azurerm_client_config" current {}
+
+    module "enterprise_scale" {
+      source = "Azure/caf-enterprise-scale/azurerm"
+      version = "0.3.1"
+
+    root_parent_id = data.azurerm_client_config.current.tenant_id
+    root_id = "contoso" 
+    root_name = "Contoso"
+    deploy_management_resources = "true"
+    subscription_id_management = "<insert_subscription_id>" 
+    configure_management_resources = {
+        settings = {
+      log_analytics = {
+        enabled = true
+        config = {
+          retention_in_days                           = 30
+          enable_monitoring_for_arc                   = true
+          enable_monitoring_for_vm                    = true
+          enable_monitoring_for_vmss                  = true
+          enable_solution_for_agent_health_assessment = true
+          enable_solution_for_anti_malware            = false
+          enable_solution_for_azure_activity          = true
+          enable_solution_for_change_tracking         = true
+          enable_solution_for_service_map             = false
+          enable_solution_for_sql_assessment          = true
+          enable_solution_for_updates                 = true
+          enable_solution_for_vm_insights             = true
+          enable_sentinel                             = true
+        }
+      }
+      security_center = {
+        enabled = true
+        config = {
+          email_security_contact             = "email@contoso.com"
```

Comment:
I'm not sure whether we should be using "potentially" real email addresses in our examples, just in case a repeat of:
https://www.zdnet.com/article/microsoft-buys-corp-com-domain-in-the-name-of-customer-security/

Reply:
Got it. Thanks for letting me know